app/backend/src/couchers/servicers/account.py

```diff
@@ -123,6 +123,16 @@ def ChangePassword(self, request, context):
 
             send_password_changed_email(user)
 
+            notify(
```

Comment:
Should these still send emails as well as the new notification?

Reply:
Yes, until the notifications are ironed out and enabled for everyone, we should still do emails.